src/site/stages/build/process-cms-exports/transformers/node-regional_health_care_service_des.js

```diff
@@ -6,15 +6,23 @@ const getFieldFacilityLocationObject = ({
   title,
   entityUrl,
   field_facility_location,
-}) =>
-  typeof title === 'object'
+}) => {
+  const fieldFacilityTitle =
+    field_facility_location && field_facility_location[0]?.title
+      ? field_facility_location[0].title
+      : '';
+  const fieldFacilityUrl =
+    field_facility_location && field_facility_location[0]?.entityUrl
+      ? field_facility_location[0].entityUrl
+      : '';
+  return typeof title === 'object'
     ? {
         entity: {
           title: getDrupalValue(title),
           fieldFacilityLocation: {
             entity: {
-              title: getDrupalValue(field_facility_location[0].title),
-              entityUrl: field_facility_location[0].entityUrl,
+              title: getDrupalValue(fieldFacilityTitle),
+              entityUrl: fieldFacilityUrl,
```

Comment:
Curous about these as well.

Reply:
This was @k80bowman's update, let me get her

Reply:
Curious too.

Looks good but the more fields needed in a future will make it harder to understand. Needed to be done in the `else` statement below as well for consistency of both ends

Reply:
There were instances in some of the test data where there was no facility title or entityUrl in that first array object of field_facility_location. So, in order to get the build to pass, I added these checks that would look for the keys in that first array object and, if it couldn't find them, set the variable to an empty string.

I don't know if this is going to be a thing when this data is published to prod, so any insights there or modifications needed are welcome. I didn't dig too much into making these changes consistent or ready for long-term use. They were purely intended to get the test builds to pass.